crates/cli/src/main.rs

```diff
@@ -54,7 +54,27 @@ fn cli() -> Command {
                 .arg(
                     arg!(-p --port <PORT> "Port to listen on")
                         .default_value("9000")
-                        .value_parser(value_parser!(u16)),
+                        .value_parser(value_parser!(u16))
+                )
+                .arg(
+                    arg!(--tls [PORT])
```

Comment:
What happens if a user provides both `--port` and `--tls` port? Can the server listen on both simultaneously?

Reply:
> What happens if a user provides both `--port` and `--tls` port? Can the server listen on both simultaneously?

yep, you can usually think of `--port` as being for http and `--tls` as being for https.

Reply:
@laktek 

If you specify the arguments as follows. edge-runtime It will listen on two ports for HTTP and TLS (HTTPS), as shown below.

Arguments
```json
[
                "start",
                "-v",
                "--tls",
                "4433",
                "--cert",
                "./localhost+1.pem",
                "--key",
                "./localhost+1-key.pem",
                "-p",
                "9998",
               // ...
            ]
```

Output
```
...
DEBUG FileFetcher::fetch_cached - specifier: https://deno.land/std@0.131.0/async/tee.ts
DEBUG Snapshot already up to date. Skipping pending resolution.
DEBUG Opening cache /home/vscode/.cache/deno/node_analysis_cache_v1...
main function started
DEBUG edge-runtime is listening on 0.0.0.0:9998
DEBUG edge-runtime is listening on 0.0.0.0:4433 (secure)
```

Reply:
https://github.com/supabase/edge-runtime/assets/46702285/998d4576-90e9-4238-9871-2da47b366cbb

Reply:
Awesome! thanks for taking time to explain how it behaves.